### DISCUSSION TITLE
How to Create Separate Projects Workflows for Issues and Pull Requests

### DISCUSSION THREAD

**Discussion by pookage** [community/community#33058] [Projects and Issues]
I just created a new repo and seen that we've been force-migrated to the new Projects system - I'm just trying to recreate the workflow automation that I had set up using the 'classic' projects now, but I'm unable to configure it so that :

- New issues have a 'backlog' status
- New pull-requests have a 'to do' status
- Merging a PR closes the linked issue and moves both to 'done'
- 'Changes requested' on a PR moves both the PR and the linked issue to 'To Do'

...This was all stuff that was trivial with the 'classic' system - can anyone indicate how to do this with the new system, or how to revert to the 'classic' system?

**Comment by revitteth** [comment]
Not sure how this doesn't have more upvotes!

**Comment by josh2318998** [comment]
To separate workflows for issues and pull requests, define their roles—issues for tasks and discussions, PRs for code reviews and merging. Automate issue labeling, assignments, and templates, while enforcing PR checks like tests and reviews. Link PRs to issues [here](https://picasoapp.com/) for tracking, use project boards for organization, and set up notifications. Monitor and refine workflows with metrics for efficiency.
